### PR TITLE
Add missing resource string to VB for Interpolated String feature.

### PR DESCRIPTION
**Pull request by AdamSpeight2008**
Fix for #17761 by adding a resource string.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -184,3 +184,4 @@ $RECYCLE.BIN/
 
 # JetBrains Rider
 .idea/
+.cr/Settings.xml
```
Comment (jmarolf) [inline .gitignore]: why this change?

Comment (AdamSpeight2008) [reply .gitignore]: Should have been in my local `.gitignore`

**Comment by AdamSpeight2008** [comment]
@dotnet-bot please retest windows_release_unit64_prtest

**Comment by AdamSpeight2008** [comment]
@dotnet-bot test windows_release_unit64_prtest please

**Comment by AdamSpeight2008** [comment]
@dotnet-bot test windows_release_vs-integration_prtest please

**Comment by CyrusNajmabadi** [comment]
Adam, as mentioned in another review, can you please give more descriptive names for your PRs?

**Comment by gafter** [comment]
@CyrusNajmabadi I think this is the first of several PRs for the US independence day.

**Comment by AdamSpeight2008** [comment]
@CyrusNajmabadi  @gafter  Are these descriptive enough? Or should I breakout the thesaurus? 📚

**Comment by AlekseyTs** [comment]
@AdamSpeight2008 Could you please re-target this change to dev15.7.x?

**Comment by AdamSpeight2008** [comment]
@AlekseyTs  Re-targeted, left the two merge conflicts as I unsure which to pick.

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review, need a second sign-off
@jaredpar For approval

**Comment by AlekseyTs** [comment]
@jaredpar For approval
